x/qgb/README.md

```diff
@@ -1,50 +1,320 @@
+
+## Events
+
+### New attestation event
+
+After creating a new attestation, and adding it to the QGB store, an event is [emitted](https://github.com/celestiaorg/celestia-app/blob/0629c757ef35a24187a8d7a4c706c7cdc894c8b6/x/qgb/keeper/keeper_attestation.go#L16-L22) containing its nonce.
+
+## Client
+
+### Query attestation command
+
+The QGB query attestation command is part of the `celestia-appd` binary. It allows the user to query specific attestations by their corresponding nonce.
+
+```shell
+$ celestia-appd query qgb attestation --help                                                     INT ✘ 
+query an attestation by nonce
+
+Usage:
+  celestia-appd query qgb attestation <nonce> [flags]
+
+Aliases:
+  attestation, att
+
+Flags:
+      --grpc-addr string   the gRPC endpoint to use for this chain
+      --grpc-insecure      allow gRPC over insecure channels, if not TLS the server must use TLS
+      --height int         Use a specific height to query state at (this can error if the node is pruning state)
+  -h, --help               help for attestation
+      --node string        <host>:<port> to Tendermint RPC interface for this chain (default "tcp://localhost:26657")
+  -o, --output string      Output format (text|json) (default "text")
+
+Global Flags:
+      --chain-id string      The network chain ID
+      --home string          directory for config and data (default "/home/midnight/.celestia-app")
+      --log-to-file string   Write logs directly to a file. If empty, logs are written to stderr
+      --log_format string    The logging format (json|plain) (default "plain")
+      --log_level string     The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace                print out full stack trace on errors
+```
+
+### Verification command
+
+The QGB verification command is part of the `celestia-appd` binary. It allows the user to verify that a set of shares has been posted to a specific QGB contract.
+
+```shell
+$ celestia-appd verify --help          
+                              
+Verifies that a transaction hash, a range of shares, or a blob referenced by its transaction hash were committed to by the QGB contract
+
+Usage:
+  celestia-appd verify [command]
+
+Available Commands:
+  blob        Verifies that a blob, referenced by its transaction hash, in hex format, has been committed to by the QGB contract.
+  shares      Verifies that a range of shares has been committed to by the QGB contract
+  tx          Verifies that a transaction hash, in hex format, has been committed to by the QGB contract
```

Comment:
[discusison around planning]
certainly not something we do in this PR, but as we've discussed in the past, we will eventually need to nail down the UX. All the tool that are needed are in these commands, but devs don't really have easy access to them. 

If they're looking at these specs, I would feel a bit confused if this were the only api, since these just logs a yes or no.